src/state/slices/actionSlice/actionSlice.ts

```diff
@@ -17,6 +17,9 @@ export const actionSlice = createSlice({
         state.byId[payload.id] = {
           ...state.byId[payload.id],
           ...payload,
+          // ensure we do *never* use payload createdAt for updates
+          // only case we should use it is for inserts
```

Comment:
should be pretty self-explanatory - not an issue currently since we don't actually do anything with `createdAt`, but spotted it while playing with validation status updates work for a given generic transaction (https://github.com/shapeshift/web/pull/9845/commits/b46931ac43ef49704c74ff623671c96d176585dd) so this fixes it while in the house!

Reply:
Nice catch on this one